commands/predict.py

```diff
@@ -69,7 +69,14 @@ def main(
     batch_size: int,
     clear_dataset_cache: bool,
 ):
-    """Compute predictions and upload them to Kili."""
+    """Compute predictions and upload them to Kili.
+
+    In order to use this command, you must first use the 'train' command.
+    This command reuses the model that is stored at the end of the training.
+    We advise you to use this command once in '--dry-run' mode.
```

Comment:
We advise you to use this command once in '--dry-run' mode to preview the predictions.